static/src/javascripts/projects/common/modules/discussion/comment-box.js

```diff
@@ -100,16 +100,19 @@ class CommentBox extends Component {
     }
 
     getDiscussionId(): string {
-        return (
-            this.options.discussionId ||
-            this.elem
-                .getAttribute('data-discussion-key')
-                .replace('discussion', '')
-        );
+        const discussionId = this.options && this.options.discussionId;
```

Comment:
Could we just return `discussionId` after this line if it exists? All the code beneath isn't required to run if `discussionId` is defined.